lib/event.py

```diff
@@ -276,6 +353,68 @@ def addComm(self, event):
         self.coidx += 1
         return True
 
+    def addComm_v2(self, event):
```

Comment:
Since we're using version control system, later we maintain only one version (not v1 & v2).

Reply:
removed all v2